### DISCUSSION TITLE
设计说明文档中有部分链接使用的是内网链接，请问这是正常的吗？

### DISCUSSION THREAD

**Discussion by Daa1874** [ant-design/ant-design#40595] [Q&A] [answered]
设计说明文档中有部分链接使用的是内网链接，请问那部分是不可查看的吗？
比如内容为设计-设计模式-模版文档-表单页中的「模板-基础表单」，访问的链接为http://preview-techui.dockerlab.alipay.net/createProject

<img width="1132" alt="image" src="https://user-images.githubusercontent.com/63888777/217195872-1b9f912a-044a-4036-bf37-2a44da45bb4c.png">

**Comment by afc163** [comment, marked as answer]
不正常，要改掉。

**Comment by zombieJ** [comment]
有问题的，需要改
